crates/cargo-test-support/src/registry.rs

```diff
@@ -166,9 +177,13 @@ pub fn init() {
 
         [registries.alternative]
         index = '{alt}'
+
+        [registries.alternative2]
+        index = '{alt2}'
```

Comment:
I think it would be best to not set this here, and only set it in the `registry_credentials` test.  Can probably just append to the config file.